Sync `svg/geometry` from WPT upstream

Ahmad-S792 opened the pull request:
#### 38eadf69e029ad55840ddf14ba6ed648e93b297f
<pre>
Sync `svg/geometry` from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=292144">https://bugs.webkit.org/show_bug.cgi?id=292144</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/5a881289df674557a02ca0b26f554f0a26b71f24">https://github.com/web-platform-tests/wpt/commit/5a881289df674557a02ca0b26f554f0a26b71f24</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-baseval-in-display-none.html::
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/circle-calc-dynamic-viewport-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/circle-calc-dynamic-viewport-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/circle-calc-dynamic-viewport.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-calc-dynamic-viewport-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-calc-dynamic-viewport-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-calc-dynamic-viewport.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/rect-calc-dynamic-viewport-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/rect-calc-dynamic-viewport-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/rect-calc-dynamic-viewport.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-get-bounding-client-rect-in-non-rendered-elements-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-get-bounding-client-rect-in-non-rendered-elements.html:

fixed

Canonical link: <a href="https://commits.webkit.org/294176@main">https://commits.webkit.org/294176@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1aaeffaba6579420a2828f89d07181d75321da66

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/101076 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/20738 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/11041 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/106229 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/51702 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/21047 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/29232 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/76995 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/34018 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/104083 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/16210 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/91298 "Passed tests") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/57342 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/16026 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/9325 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/51050 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/85931 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/9381 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/108578 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/28204 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/20766 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/85955 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/28566 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/87498 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/85493 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/21753 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/30215 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/7942 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/22294 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/28134 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/27946 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/31266 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/29504 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/3ed254582e98908dbb5f9abfc875d9c220198658. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/44587

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/1aaeffaba6579420a2828f89d07181d75321da66. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/44587

Commit:
Sync `svg/geometry` from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=292144

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/5a881289df674557a02ca0b26f554f0a26b71f24

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-baseval-in-display-none.html::
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/circle-calc-dynamic-viewport-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/circle-calc-dynamic-viewport-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/circle-calc-dynamic-viewport.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-calc-dynamic-viewport-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-calc-dynamic-viewport-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/ellipse-calc-dynamic-viewport.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/rect-calc-dynamic-viewport-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/rect-calc-dynamic-viewport-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/reftests/rect-calc-dynamic-viewport.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-get-bounding-client-rect-in-non-rendered-elements-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/geometry/svg-get-bounding-client-rect-in-non-rendered-elements.html:

fixed

Canonical link: https://commits.webkit.org/294176@main
